lib/resources/dstu2/observation.yaml

```diff
@@ -0,0 +1,129 @@
+                 observations.
+  example: |
+             {"component": [
+              {"code": {
+                 "coding": [
+                  {"system": "http://loinc.org",
+                   "code": "8480-6",
+                   "display": "INTRAVASCULAR SYSTOLIC:PRES:PT:ARTERIAL SYSTEM:QN:"}],
+                 "text": "Systolic Blood Pressure"},
+               "valueQuantity": {
+                 "value": 128,
+                 "unit": "mmHg",
+                 "system": "http://unitsofmeasure.org",
+                 "code": "mm[Hg]"},
+               "referenceRange": [
+                {"low": {
+                   "value": 46,
+                   "unit": "mmHg",
+                   "system": "http://unitsofmeasure.org",
+                   "code": "mm[Hg]"},
+                 "high": {
+                   "value": 97,
+                   "unit": "mmHg",
+                   "system": "http://unitsofmeasure.org",
+                   "code": "mm[Hg]"}}]},
+              {"code": {
+                 "coding": [
+                  {"system": "http://loinc.org",
+                   "code": "8462-4",
+                   "display": "INTRAVASCULAR DIASTOLIC:PRES:PT:ARTERIAL SYSTEM:QN:"}],
+                 "text": "Diastolic Blood Pressure"},
+               "valueQuantity": {
+                 "value": 82},
+               "referenceRange": [
+                {"low": {
+                   "value": 44},
+                 "high": {
+                   "value": 123}}]}]
+  children:
+
+  - name: code
+    required: 'Yes'
+    cardinality: 1..1
+    type: CodeableConcept
+    description: Describes what was observed. Sometimes this is called the observation "code".
+    example: |
+               {"code": {
+                  "coding": [
+                   {"system": "http://loinc.org",
+                    "code": "8480-6",
+                    "display": "INTRAVASCULAR SYSTOLIC:PRES:PT:ARTERIAL SYSTEM:QN:"}],
+                  "text": "Systolic Blood Pressure"}}
+    binding:
+      description: Codes identifying names of simple observations.
+      terminology:
+      - display: LOINC
+        system: http://loinc.org
+
+- name: interpretation
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: >
+                 The assessment made based on the result of the observation. Intended as a simple compact code often
+                 placed adjacent to the result value in reports and flow sheets to signal the meaning/normalcy status
+                 of the result. Otherwise known as abnormal flag.
+  example: |
+             {"interpretation": {
+                "coding": [
+                 {"system": "http://hl7.org/fhir/v2/0078",
+                  "code": "H",
+                  "display": "High"}],
+                "text": "HI"}}
+  binding:
+    description: Codes identifying interpretations of observations.
+    terminology:
+    - display: Version 2 Table 0078
```

Comment:
Would "v2 Interpretation Codes" be more descriptive? Otherwise I think this should be "**HL7** Version 2 Table 0078"